### PR TITLE
Update restore-deleted-group.md

### PR DESCRIPTION
**Pull request by rykoma**
The descriptions of the permission were wrong, so I fixed it.

When restoring the group using EAC, the user must be an administrator.
When restoring the group using AzureADPreview command, if the user is an administrator, the user can restore all deleted groups. Even if the user is not an administrator, the user can restore the group which the user owns.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 1]
```diff
@@ -49,9 +49,14 @@ When a group is restored, the following content is restored:
 - Yammer group and group content (If the Office 365 group was created from Yammer)
     
 You can also [Permanently delete an Office 365 group](#permanently-delete-an-office-365-group) if you can't wait the 30 days for the retention period to expire for the content to be permanently deleted. 
-  
+
+> [!NOTE]
+> The owner of the deleted Office 365 group also be able to restore the group. To restore an office 365 Group using owner permission without administrator permission, see [Restore an Office 365 Group using PowerShell](#restore-an-office-365-group-using-powershell).
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]: The owner of the deleted Office 365 group **is also** able to restore the group. To restore an office 365 **group** using owner permission without administrator permission, see [Restore an Office 365 Group using PowerShell](#restore-an-office-365-group-using-powershell).

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 2]
```diff
@@ -88,11 +93,15 @@ You can also [Permanently delete an Office 365 group](#permanently-delete-an-off
     
 ## Restore an Office 365 Group using PowerShell
 
-IMPORTANT: If you use Remove-MsolGroup in PowerShell to delete a group, this will delete the group permanently. When using PowerShell to delete groups, it's best practice to use **Remove-AzureADMSGroup** to soft-delete the Office 365 group. That way you can restore it if needed. 
+You must have Office 365 global administrator permissions, or need to be an owner of the deleted Office 365 Group.
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]: You must have Office 365 global administrator **permission**, or need to be an owner of the deleted Office 365 group.

Please do changes as indicated.

File: Office365-Admin/create-groups/restore-deleted-group.md (hunk 1 above)
Comment (rykoma) [reply Office365-Admin/create-groups/restore-deleted-group.md]: I updated that point. Could you review it again please?

File: Office365-Admin/create-groups/restore-deleted-group.md (hunk 2 above)
Comment (rykoma) [reply Office365-Admin/create-groups/restore-deleted-group.md]: I updated that point. Could you review it again please?

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ You must have Office 365 global administrator or Exchange administrator permissi
     
 ## Restore an Office 365 Group using PowerShell
 
-You must have Office 365 global administrator permissions, or need to be an owner of the deleted Office 365 Group.
+You must have Office 365 global administrator permission, or need to be an owner of the deleted Office 365 group.
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]: Copy edit - no changes.

Comment (rykoma) [reply Office365-Admin/create-groups/restore-deleted-group.md]: What do you mean? Is everything ok now?

Comment (JohanFreelancer9) [reply Office365-Admin/create-groups/restore-deleted-group.md]: Yes everything is ok.

**Comment by rykoma** [comment]
@mypil Is it possible to proceed this PR?

**Comment by mypil** [comment]
@rykoma - We could not proceed yet since we were notified by the O365 content writers that there's a major milestone that requires them to make a lot of changes to the articles in the Office365-Admin section of our help set so this includes this article where you submitted a PR on.  This means when a customer submits a change, they want to hold off on doing anything about it until after they notify us that everything is in the clear.

This may take a few weeks but rest assured that as soon as this will be lifted then we will immediately proceed on the regular routine of tagging the doc owners to review and merge the content of PRs in this repository.

We hope for your patience and kind understanding. Thank you.

**Comment by dianef77** [comment]
@dianef77 please review and decide

**Comment by mypil** [comment]
@dianef77 - Do you have any updates on this?

Thank you.

**Comment by rykoma** [comment]
@mypil, who is responsible for this repository?

**Comment by mypil** [comment]
@rykoma - we are coordinating with the doc owner @dianef77 or @MikePlumleyMSFT to get their approval to merge this PR your created.

@MikePlumleyMSFT - would you be able to help review and merge the content of this PR?

cc: @pamgreen @kenwith 

Thank you.

**Comment by mypil** [comment]
@MikePlumleyMSFT - I would like to follow up on this. Can you please assist to check, approve and merge the content of this PR?

Thank you.

**Pull request merged**